BaseListener/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#install zip on debian OS, since microsoft/dotnet container doesn't have zip by default
+if [ -f /etc/debian_version ]
+then
+  apt -qq update
+  apt -qq -y install zip
+fi
+
+#dotnet restore
+dotnet tool install --global Amazon.Lambda.Tools --version 4.0.0
+
+
+# (for CI) ensure that the newly-installed tools are on PATH
+if [ -f /etc/debian_version ]
+then
+  export PATH="$PATH:/$(whoami)/.dotnet/tools"
+fi
+
+dotnet restore
+dotnet lambda package --configuration release --framework netcoreapp3.1 --output-package ./bin/release/netcoreapp3.1/base-listener.zip
```

Comment:
No.
I have modified the Renamer.ps1 file so that it is called with an additional parameter which replaces all instances of `base-listener`. See the readme in the root of the repo for instructions.